lib/src/over_react_test/common_component_util.dart

```diff
@@ -335,13 +334,13 @@ void _testPropForwarding(BuilderOnlyUiFactory factory, dynamic childrenFactory()
       /// Test for prop keys that both are forwarded and exist on the forwarding target's default props.
       if (isDartComponent(forwardingTarget)) {
         final forwardingTargetType = (forwardingTarget as ReactElement).type as ReactClass;
-        Map forwardingTargetDefaults;
+        late Map forwardingTargetDefaults;
         switch (forwardingTargetType.dartComponentVersion) { // ignore: invalid_use_of_protected_member
           case ReactDartComponentVersion.component: // ignore: invalid_use_of_protected_member
-            forwardingTargetDefaults = forwardingTargetType.dartDefaultProps; // ignore: deprecated_member_use
+            forwardingTargetDefaults = forwardingTargetType.dartDefaultProps ?? {}; // ignore: deprecated_member_use
```

Comment:
In this case we can assume these are non-null (and if they're not for some reason, I think it'd be better to error  than proceed silently)
```suggestion
            forwardingTargetDefaults = forwardingTargetType.dartDefaultProps!; // ignore: deprecated_member_use
```